app/views/pages/_coc_text.html.erb

```diff
@@ -31,7 +31,7 @@
 </ul>
 <h2>Enforcement</h2>
 <p>Violations of the Code of Conduct may be reported by contacting the team via the
-  <a href="<%= app_url("/report-abuse") %>">abuse report form</a> or by sending an email to <a href="mailto:<%= SiteConfig.email_addresses[:default] %>" rel="noopener noreferrer"><%= SiteConfig.email_addresses[:default] %></a>. All reports will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. Further details of specific enforcement policies may be posted separately.
```

Comment:
After some research and according to some [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) we only need an rel="noopener noreferrer" when there is a target=_blank on the `a` tag. For this reason I did not cater for rel="noopener noreferrer". If anyone knows any differently please let me know.